pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,3 +47,46 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 		MountPoints: mountPoints,
 	}, nil
 }
+
+// GetPodResourceRequest returns the pod requests for the supported resources.
+// Pod request is the summation of resource requests of all containers in the pod.
+func GetPodResourceRequests(pod *api.Pod) api.ResourceList {
+	requests := api.ResourceList{}
+	for _, container := range pod.Spec.Containers {
+		requests = quota.Add(requests, container.Resources.Requests)
```

Comment:
I was discussing this with @smarterclayton and @deads2k recently that they could move elsewhere.  I am fine to move those in a follow-on (basically, everything but `CalculateUsage`) in that resources.go.  I can look to do that.